cves/CVE-2013-4310.yml

```diff
@@ -189,8 +201,10 @@ lessons:
     applies:
     note:
   yagni:
-    applies:
-    note:
+    applies: true
+    note: |
+      Only allow security access to features that need it. Only use prefix where
+      needed to eliminate special case vulnerabilities.
```

Comment:
Is this supposed to be under "least privilege"? For some reason it looks like those lines in the yaml file got deleted and it appears that this comment is under the YAGNI principle, which I don't believe applies in this situation.

Reply:
I agree, it looks like this accidentally got put under YAGNI

Reply:
Oops. I have moved the note back under 'least privilege' where it belongs.